### PR TITLE
chore: Update admin/userguide

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
See https://github.com/codeigniter4/userguide/pull/37#issuecomment-1370402918

- copy the latest repo-specific files from `codeigniter4/userguide`
- update deploy script

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [ ] User guide updated
- [ ] Conforms to style guide

### CODE REVIEW

File: .github/scripts/deploy-userguide [hunk 1]
```diff
@@ -23,6 +23,12 @@ cd $TARGET
 git checkout master
 rm -rf docs
 
+# Copy common files
+cp -Rf ${SOURCE}/LICENSE ./
+
+# Copy repo-specific files
+cp -Rf ${SOURCE}/admin/userguid/. ./
```
Comment (paulbalandan) [inline .github/scripts/deploy-userguide:30]:
```suggestion
cp -Rf ${SOURCE}/admin/userguide/. ./
```

File: admin/userguide/.gitignore [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+user_guide_src/build/*
+user_guide_src/cilexer/build/*
+user_guide_src/cilexer/dist/*
+user_guide_src/cilexer/pycilexer.egg-info/*
+
+#codeigniter 3
+application/logs/*
+!application/logs/index.html
+!application/logs/.htaccess
```
Comment (paulbalandan) [inline admin/userguide/.gitignore:16]: Do we need these?

File: .github/scripts/deploy-userguide (hunk 1 above)
Comment (kenjis) [reply .github/scripts/deploy-userguide]: Oops. Fixed. Thanks!

File: admin/userguide/.gitignore (hunk 2 above)
Comment (kenjis) [reply admin/userguide/.gitignore]: No. Removed.

**Comment by kenjis** [comment]
The source for GitHub Pages is `master` `/docs`:
Screenshot 2023-01-07 10 15 25

So I think the file https://github.com/codeigniter4/userguide/blob/master/.nojekyll is not necessary.
I did not copy it to `admin/userguide`.

**Review by MGatner — APPROVED** [review]
👍

**Review by paulbalandan — APPROVED** [review]

**Pull request merged**